### ISSUE TITLE
Missing bins not added when binning reads and running in stranded mode

### ISSUE THREAD

**Issue by pjuren** [smithlabcode/piranha#4]
When Piranha is told to bin reads, it is supposed to add the missing bins to the covariates. This worked in the previous version:

![screen shot 2015-05-14 at 3 27 33 pm](https://cloud.githubusercontent.com/assets/7141049/7643397/16751486-fa4e-11e4-9133-f191e734b8c6.png)

and continues to work in the new version when strandedness is disabled:

![screen shot 2015-05-14 at 3 31 52 pm](https://cloud.githubusercontent.com/assets/7141049/7643453/72afc8ea-fa4e-11e4-8dac-d7519ae4719d.png)

But fails without the -x option:

![screen shot 2015-05-14 at 3 33 40 pm](https://cloud.githubusercontent.com/assets/7141049/7643462/98e1a862-fa4e-11e4-9bb0-4b8dbc2eaadb.png)

Basically making the program unusable for anyone wanting to analyze RIP-seq data (!!)

**Comment by bahramis** [comment]
Fixed.
